src/explore-education-statistics-admin/src/components/ReleaseServiceStatus.tsx

```diff
@@ -109,6 +109,11 @@ const ReleaseServiceStatus = ({
               ? currentStatus.overallStage
               : 'Waiting to be scheduled...'
           }
+          id={
+            currentStatus
+              ? `ReleaseProcessStatus-${currentStatus.overallStage}`
```

Comment:
Would recommend using mixture of camcelCase/kebab-case for ids. This would be consistent with most other ids we have.